LICENSE

```diff
@@ -23,7 +23,7 @@
       "You" (or "Your") shall mean an individual or Legal Entity
       exercising permissions granted by this License.
 
-      "Source" form shall mean the preferred form for making modifications,
```

Comment:
This is Apache license sentence so that we can't modify this. Please revert this commit.